### PR TITLE
refactor(picturebook): Switch to a sane approach

### PR DESCRIPTION
**Pull request by obartra**
Change to a utility method based approach. New methods:

- getFiles
- loadStories
- nightwatchConfig
- compareImages
- runTests
- runTestsWithTunnel

BREAKING CHANGE: Picturebook 2.0 is a complete rewrite. No APIs remained consistent

### CODE REVIEW

**Comment by arahansen** [comment]
lol

image

**Comment by obartra** [comment]
293 of them were in `flow-typed`!

File: README.md [hunk 1]
```diff
@@ -1,93 +1,251 @@
...
+  +platform: string,
+  +browser: string
+|}
+
+function getFiles(userOptions: $Shape<Options>): Array<StoryPaths>
+
+function loadStories(userOptions: $Shape<Options>): Array<LoadedStory>
+
+function nightwatchConfig(params: {
+  desiredCapabilities?: {},     // nightwatch desired capabilities object
```
[the remark this answers is not in the document]
Comment (obartra) [reply README.md]: https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities

File: README.md [hunk 2]
```diff
@@ -1,93 +1,251 @@
...
+```
 
-Simply specify a `storyPath` on your config (e.g. `src/stories`) and PictureBook will take care of the rest. The name of the story is set by un-camel-casing the file name.
+Where `screenshots` is the output of `runTestsWithTunnel` or `runTests` (if running with or without a tunnel, respectively).
 
-Documentation is added to each story if there's a matching file with the same name but with `md` extension.
+## 🙋 QYMA (Questions you may ask)
 
-To add actions (using `@storybook/addon-actions`) follow [these usage instructions](https://github.com/storybooks/storybook/tree/master/addons/actions).
+### Why do I need to add localtest.dev as a localhost alias?
```
[the remark this answers is not in the document]
Comment (obartra) [reply README.md]: this is needed any time you want to use the tunnel with localhost and you want to test safari or ie11

**Pull request merged**